src/main/java/com/strandls/observation/util/ObservationUtilModule.java

```diff
@@ -0,0 +1,12 @@
+package com.strandls.observation.util;
+
+import com.google.inject.AbstractModule;
+import com.google.inject.Scopes;
+
+public class ObservationUtilModule extends AbstractModule {
+
+	@Override
+	protected void configure() {
+		bind(TokenGenerator.class).in(Scopes.SINGLETON);
```

Comment:
include it directly inside context listerner class, we can save creation of a new class then